Automattic/wp-calypso
# Stats: Adding UTM type dropdown

Related to #87573

## Proposed Changes

* adding a dropdown to allow switching between displayed data in the module

## Testing Instructions



* open a live branch
* apply `stats/utm-module` feature flag
* verify that the dropdown is visible next to the header
* verify that you can change the dropdown options and its reflected in the header of above the bars

![SCR-20240304-nqhu](https://github.com/Automattic/wp-calypso/assets/112354940/01275720-3e98-40fd-b531-44309e7310e2)

## Pre-merge Checklist
